### PR TITLE
chore: migrate create database view to chakra

### PR DESCRIPTION
**Pull request by pradel**

### CODE REVIEW

File: client/src/pages/create-database.tsx [hunk 1]
```diff
@@ -93,27 +102,27 @@ export const CreateDatabase = () => {
...
-            'Name already exists',
-            `You already have created ${type} database with this name`,
-            (name) =>
-              !dataDb?.databases.find(
-                (db) => db.name === name && type === db.type
-              )
-          );
-      }),
+      .matches(/^[a-z0-9-]+$/),
+    // .when('type', (type: DatabaseTypes) => {
```
Comment (pradel) [inline client/src/pages/create-database.tsx:114]: @Akirtovskis this part is actually breaking the validation, by commenting it we check that the field is not empty and match the regex. Do you have an idea why by any chance?

**Comment by pradel** [comment]
Except for the validation issue (not related to this pr originally but can be good to fix at the same time), this is good to go

File: client/src/pages/create-database.tsx (hunk 1 above)
Comment (Akirtovskis) [reply client/src/pages/create-database.tsx:114]: @pradel not immediate idea, would have to try this on fresh branch. Let's merge this PR with this part commented out and create an issue to return to this later?

Comment (Akirtovskis) [reply client/src/pages/create-database.tsx:114]: @pradel not immediate idea, will check now on a fresh branch.

Comment (Akirtovskis) [reply client/src/pages/create-database.tsx:114]: @pradel I will create issue as this seems like an issue multiple people are facing here : https://github.com/jquense/yup/issues/851 . Definitely there should be a way around, but let's leave this part commented out for now.

Comment (Akirtovskis) [reply client/src/pages/create-database.tsx:114]: This worked : 

```
 name: yup.string().when('type', (type: DatabaseTypes) => {
      return yup
        .string()
        .required('Database name is required')
        .matches(/^[a-z0-9-]+$/)
        .test(
          'Name already exists',
          `You already have created ${type} database with this name`,
          (name) =>
            !dataDb?.databases.find(
              (db) => db.name === name && type === db.type
            )
        );
    }),
```

**Pull request merged**